### PR TITLE
Add handler to get player version from the module.

### PR DESCRIPTION
**Pull request by enjikaka**
Add handler to get player version from the module.

### CODE REVIEW

File: packages/player/src/api/index.ts [hunk 1]
```diff
@@ -4,15 +4,16 @@ export { getMediaProduct } from '../internal/handlers/get-media-product';
...
-export { getVolumeLevel } from '../internal/handlers/get-volume-level';
 
+export { getVolumeLevel } from '../internal/handlers/get-volume-level';
 export { load } from '../internal/handlers/load';
 export { pause } from '../internal/handlers/pause';
 export { play } from '../internal/handlers/play';
 export { reset } from '../internal/handlers/reset';
-export { seek } from '../internal/handlers/seek';
 
+export { seek } from '../internal/handlers/seek';
```
Comment (osmestad) [inline packages/player/src/api/index.ts:16]: I don't get why `seek` is moved to this group?

Comment (enjikaka) [reply packages/player/src/api/index.ts]: Think I got some fun leftover when the formatter moved my line to above load... Fixed in f4fddd19215d737977a30928e3127d287392edc0

File: packages/player/.env.production [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Automatically generated from package.json
```
Comment (osmestad) [inline packages/player/.env.production:1]: Should this file be in `git` or not?

Comment (enjikaka) [reply packages/player/.env.production:1]: Should be safe to gitignore I guess. As long as we make sure to run the script before Vite. I'll update.

**Pull request merged**